pkg/specgen/specgen.go

```diff
@@ -43,6 +43,13 @@ type ContainerBasicConfig struct {
 	// image's configuration.
 	// Optional.
 	Command []string `json:"command,omitempty"`
+	// EnvHost indicates that the host environment should be added to container
+	// Optional.
+	EnvHost bool `json:"env_host,omitempty"`
```

Comment:
Why are we sourcing this from the server, instead of the host? This doesn't make sense, IMO

Reply:
If you are on a MAC or Windows box you want this from the server.  The Proxies are setup on the Server, Proxies on the Client might not work while you are on the server machines.